robert_beverage/scripts/generate_qr_code

```diff
@@ -1,8 +1,27 @@
 #!/usr/bin/env python
+import rospy
 import os
 import qrencode
 import argparse
+import tf2_ros
 
+rospy.init_node('qr_code_generator')
+tfBuffer = tf2_ros.Buffer()
+listener = tf2_ros.TransformListener(tfBuffer)
+rate = rospy.Rate(10.0)
+
+def get_coordinates():
+    while True:
```

Comment:
Endless loop in general is not good idea. Please use `not rospy.is_shutdown()` function as condition of the loop as it is specified here http://wiki.ros.org/rospy/Overview/Initialization%20and%20Shutdown#Testing_for_shutdown